common/rya.api/src/main/java/org/apache/rya/api/instance/RyaDetailsToConfiguration.java

```diff
@@ -53,14 +53,16 @@ public static void addRyaDetailsToConfiguration(final RyaDetails details, final
         checkAndSet(conf, ConfigurationFields.USE_FREETEXT, details.getFreeTextIndexDetails().isEnabled());
       //RYA-215        checkAndSet(conf, ConfigurationFields.USE_GEO, details.getGeoIndexDetails().isEnabled());
         checkAndSet(conf, ConfigurationFields.USE_TEMPORAL, details.getTemporalIndexDetails().isEnabled());
-        PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
-		if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
-			checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
-			conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
-			conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
-		} else {
-			checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, false);
-		}
+        final PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
+        if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
+            checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
+            conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
+            conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
+            conf.set(ConfigurationFields.PCJ_STORAGE_TYPE, "ACCUMULO");
```

Comment:
So this is probably okay for now.  But this method could take in either an instance of AccumuloRyaInstanceDetails or an instance of MongoRyaInstanceDetails.  So the storage type could be Accumulo or Mongo (note that the Mongo storage type doesn't currently exist in master, but will be merged in as part of a pull request).  Given that there is currently no updater for mongo, there will be no fluo details so the the Accumulo storage type will never be set for a Mongo backed instance.  But this could change once an updater is implemented for Mongo.  The best thing to do is to add a getStorageType() method to the RyaInstanceDetails, and then just pass this into the config so that the storage type aligns with the RyaInstanceDetails type.